consensus/src/chained_bft/event_processor_fuzzing.rs

```diff
@@ -23,7 +23,7 @@ use libra_types::crypto_proxies::{LedgerInfoWithSignatures, ValidatorSigner, Val
 use network::{proto::Proposal, validator_network::ConsensusNetworkSender};
 use once_cell::sync::Lazy;
 use prost::Message as _;
-use safety_rules::{InMemoryStorage, SafetyRules};
+use safety_rules::{PersistentStorage as SafetyStorage, SafetyRules};
```

Comment:
probably just rename in safety_rules 😉

Reply:
Lemme do a follow up commit so I don't make this too unwieldy